ballerina/tests/mock_test.bal

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+
+final Client mockClient = check new Client(config, serviceUrl = "http://localhost:9090/crm/v3/objects/meetings");
+
+@test:Config {}
+function testMockUpsertBatch() returns error? {
+    BatchInputSimplePublicObjectBatchInputUpsert payload = {
+        "inputs": [
+            {
+                "properties": {},
+                "associations": [],
+                "id": "21212121121"
+            }
+        ]
+    };
+    BatchResponseSimplePublicUpsertObject|BatchResponseSimplePublicUpsertObjectWithErrors response = check mockClient->/batch/upsert.post(payload);
```

Comment:
Line shouldn't exceed 120 character. Let's break this line into 2.
Check other places and do the same.